src/open_samus_returns_rando/specific_patches/static_fixes.py

```diff
@@ -338,14 +338,12 @@ def disable_vignettes(editor: PatcherEditor) -> None:
                 for sg in sg_group:
                     # Check for the cc_name
                     if sg["str1"] == cc_name:
-                        for cc_group in range(len(sg["struct3"])):
-                            model_group = sg["struct3"][cc_group]["struct5"]
-                            for model in model_group:
-                                idx = model_group.index(model)
-                                for vignette in vignette_models:
+                        for cc_group in sg["struct3"]:
+                            model_group = cc_group["struct5"]
+                            for idx, model in reversed(list(enumerate(model_group))):
```

Comment:
can't this be chaged to `... reversed(list(model_group))`?

Reply:
I want indices of the original list so `reversed` must be there. and you can't `reversed` and `enumerate`.